english/bear/bear.md

```diff
@@ -2,4 +2,8 @@ Stella atroir.
 Taiwan bear.
 Ice cold. If warm, the results could be unbearable.
 Bear in mind, the grizzly taste.
-You are now drunk. You stumble over to the nearest cutie and ask, "Hey baby, what's ursine?"
\ No newline at end of file
+You are now drunk. You stumble over to the nearest cutie and ask, "Hey baby, what's ursine?"
+
+When she slaps you in the face...
+
+[You vomit on top of her](english/vomit/vomit.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.